packages/vuepress-theme-nimatron/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@okta/vuepress-theme-nimatron",
```

Comment:
Isn't the name `nimatron` deprecated?

Reply:
It is, because we're really treating this as a deprecated/legacy theme. In the (near) future we will be rolling out `vuepress-theme-odyssey`